src/pytest/test.py

```diff
@@ -913,13 +910,13 @@ def testExact(env):
 
 def testGeoErrors(env):
     env.expect('flushall')
-    env.expect('ft.create idx schema name text location geo').equal('OK')
+    env.expect('ft.create idx ON HASH FILTER startswith(@__key,"") schema name text location geo').equal('OK')
     env.expect('ft.add idx hotel 1.0 fields name hill location -0.1757,51.5156').equal('OK')
     env.expect('ft.search idx hilton geofilter location -0.1757 51.5156 1 km').equal([0L])
 
-    # Insert error
-    env.expect('ft.add', 'idx', 'hotel1', 1, 'fields', 'name', '_hotel1', 'location', '1, 1').error()   \
-            .contains('Could not index geo value')
+    # Insert error - works fine with out of keyspace implementation
```

Comment:
?

Reply:
Error was raised b/c of space between `long` and `lat` causes redis an error.
The current parsing which does not use Redis GEO handle this correctly.
In addition, currently, `ft.add` does not get errors from `HSET`